Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml

```
@@ -450,6 +450,47 @@
   - type: BorgModuleIcon
     icon: { sprite: Interface/Actions/actions_borg.rsi, state: wire-module }
 
+- type: entity
+  id: BorgModuleArtistry
+  parent: [ BaseBorgModule, BaseProviderBorgModule ]
+  name: artistry cyborg module
+  description: A module for arts & crafts whilst the station burns!
+  components:
+  - type: Sprite
+    layers:
+    - state: generic
+    - state: icon-artistry
+  - type: ItemBorgModule
+    hands:
+    - item: SprayPainter
+      hand:
+        emptyRepresentative: SprayPainterEmpty
+        emptyLabel: empty
+        whitelist:
+          tags:
+          - SprayPainter
```

You don't need to create new tags, you can specify the components in the whitelist. With SprayPainterAmmo also
```suggestion
        whitelist:
          components:
          - SprayPainter
```